docs/sources/flow/reference/components/module.string.md

```diff
@@ -0,0 +1,136 @@
+---
+title: module.string
+---
+
+# module.string
+
+`module.string` is a *Module loader* component. A *Module loader* is a Grafana Agent Flow 
```

Comment:
I don't think it's a proper noun. 

Also, after the first usage you can drop the italics.

```suggestion
`module.string` is a *module loader* component. A module loader is a Grafana Agent Flow 
```